packages/dominus-settings/settingsCapitals.js

```diff
@@ -29,7 +20,16 @@ _gs.capitals = function(gameId, path) {
       glass: 80
     }
 
+    settings.startingGarrison = {
+      footmen: 24,
+      archers: 24,
+      pikemen: 24,
+      cavalry: 0,
+      catapults: 0
+    }    
+
     settings.incomeInterval = 1000 * 60 * 10;
+    settings.villagePercentageIncome = 0.12;
```

Comment:
Ugh, seems like https://github.com/dan335/dominus/pull/2/commits/361ad406c333e4ab9eaa8bc768933f57453de006#diff-1d09e0b2953b183777c26e43b4b3b42aR33 is breaking income job. Reverting back in a minute.